src/core/link-to-dfn.js

```diff
@@ -24,13 +25,12 @@ export function run(conf, doc, cb) {
         if (oldIsDfn && newIsDfn) {
           // Only complain if the user provides 2 <dfn>s
           // for the same term.
-          pub(
-            "error",
-            "Duplicate definition of '" +
-              (dfn_for ? dfn_for + "/" : "") +
-              title +
-              "'"
-          );
+          dfn.addClass("respec-offending-element");
+          dfn.attr("title", "This is defined more than once in the document");
```

Comment:
Just realized we should localize this string. At the top of the file:

```JS
import { lang as defaultLang } from "../l10n";
const l10n = {
  en: {
     "duplicate": "This is defined more than once in the document."
  }
};

// Fall back to English, if language is missing
const lang = defaultLang in l10n ? defaultLang : "en";
```

And then here:

```JS
dfn.attr("title", l10n[defaultLang].duplicate);
```

Hope that makes sense!

Reply:
Yes it makes sense